### PR TITLE
Chaged docker port from 2475 to back to 2375

### PR DESCRIPTION
**Pull request by wulfthimm**
## Description

We are currently running two docker installtions in `core-dev`, `docker-engine` and `docker-engine-update`. The latter one has been added to unblock Gitpod deployments as should have been a temporary solution. But after upgrading the first one we forgot to do so. This changes the port to use the `docker-engine` again. After merging we could remove the other one after all branches are using docker on port 2375.

## Related Issue(s)

Fixes #

## How to test

`werft` builds should work as before like [this job](https://werft.gitpod-dev.com/job/gitpod-build-wth-change-docker-port.0).
## Release Notes

```release-note
None
```

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **metcalfc** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @metcalfc` in a comment when ready.

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by mads-hartmann — APPROVED** [review]

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: 3cd5c4c714506e29555196486a3b777ca8b7d39e</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/7948#pullrequestreview-870237878" title="Approved">mads-hartmann</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **metcalfc** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @metcalfc` in a comment when ready.

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by mads-hartmann** [comment]
@wulfthimm Added the hold label, lets get https://github.com/gitpod-io/ops/pull/919 merged and applied first before we start using it :)

**Comment by wulfthimm** [comment]
> @wulfthimm Added the hold label, lets get [gitpod-io/ops#919](https://github.com/gitpod-io/ops/pull/919) merged and applied first before we start using it :)

I applied the changes in [Argo-CD](https://argo-cd.gitpod-io-dev.com/applications/gitpod-core-dev-docker-engine?resource=). We are ready to merge.

**Comment by mads-hartmann** [comment]
@wulfthimm Sounds good. Just remove the hold label and add the approved label whenever you think is a good time to merge.

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

Approval requirements bypassed by manually added approval.

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/7948#pullrequestreview-870237878" title="Approved">mads-hartmann</a>*

*No associated issue*. Requirement bypassed by manually added approval.

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
